src/test/extension.test.ts

```diff
@@ -1,104 +1,169 @@
+import { createStubExtensionContext, createStubSourcegraphAPI } from '@sourcegraph/extension-api-stubs'
 import expect from 'expect'
+import { uniqueId } from 'lodash'
 import mock from 'mock-require'
-import { createMockSourcegraphAPI, projects } from './stubs'
 
-const sourcegraph = createMockSourcegraphAPI()
-// For modules importing Range/Location/URI/etc
-mock('sourcegraph', sourcegraph)
+import { activate, decorateEditor, decorateLine, getDecorations } from '../extension'
+import { SentryProject } from '../settings'
 
-import { decorateEditor, decorateLine, getDecorations } from '../extension'
+export const sourcegraph = createStubSourcegraphAPI()
+// For modules importing Range/Location/Position/URI/etc
+mock('sourcegraph', sourcegraph)
```

Comment:
It's important that you mock the `sourcegraph` module before importing your extension (else the extension will import `sourcegraph` before the stubs are registered)